samples/bluetooth/mesh/light_ctrl/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -14,21 +14,13 @@ CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL=y
 CONFIG_MCUMGR_CMD_IMG_MGMT=y
 CONFIG_MCUMGR_CMD_OS_MGMT=y
 
-# Enable MCUmgr Packet Reassembly feature over Bluetooth and its
```

Comment:
I have not modified these files, and suspect they might've been changed automatically

Reply:
I think I know what the issue is. Your fork version of the main branch is not up to date with the NCS main branch, hence the diff. This can be solved by fetching the main branch here:
![image](https://user-images.githubusercontent.com/35562897/177487409-24e269da-5a94-4b0b-bf51-79b66cca81b5.png)

Reply:
These changes was added last friday I see by Aleksander Nowakowski:
![image](https://user-images.githubusercontent.com/35562897/177488631-741abdf4-6e1c-46b2-afa8-fa0330341014.png)